docs/source/conf.py

```diff
@@ -63,7 +63,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path .
-exclude_patterns = ['_build','build',]
+exclude_patterns = ['_build', 'build', ]
```

Comment:
Trailing `,` looks odd here. Not sure if that could cause problems or not.

Reply:
Looks like it's okay... I wondered if it would create a null or _empty_ value at index 2. Looks like it doesn't.